features/delayed_job.feature

```diff
@@ -2,7 +2,7 @@ Feature: Bugsnag detects errors in Delayed job workers
 
 Background:
   Given I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
-  And I set environment variable "MAZE_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
+  And I set environment variable "BUGSNAG_API_KEY" to "a35a2a72bd230ac0aa0f52715bbdc6aa"
```

Comment:
Should this line be removed? It looks like the same step as the previous line. Same question/comment applies to a few other .feature files below.